modules/creating-logfilesmetricexporter.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-logfilesmetricexporter_{context}"]
+= Creating a LogFileMetricExporter resource
+
+In {logging} version 5.8 and newer versions, the `LogFileMetricExporter` custom resource (CR) is no longer deployed with the collector by default.
+You must manually create a `LogFileMetricExporter` CR to generate metrics from the logs produced by running containers.
+
+If you do not create the `LogFileMetricExporter` CR, you may see a *No datapoints found* message in the {product-title} web console dashboard for *Produced Logs*.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {clo}.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Create a `LogFileMetricExporter` CR as a YAML file:
++
+.Example `LogFileMetricExporter` CR
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogFileMetricExporter
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  nodeSelector: # <1>
+    key1: value1
+  resources: # <2>
+    limits:
+      cpu: 500m
+      memory: 256Mi
+    requests:
+      cpu: 200m
+      memory: 128Mi
+  tolerations: # <3>
+    - key: collector
+      operator: Exists
+      effect: NoSchedule
+      tolerationSeconds: 6000
+# ...
+----
+<1> The `nodeSelector` syntax defines which nodes the pods are scheduled on.
+<2> The `resources` syntax defines resource requirements for the `LogFileMetricExporter` CR.
+<3> The `tolerations` syntax defines the tolerations that the pods accept.
+
+. Apply the `LogFileMetricExporter` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+.Verification
+
+A `logfilesmetricexporter` pod runs concurrently with a `collector` pod on each node.
+
+If this is not occurring due to incorrect settings for the `tolerations` syntax or the `nodeSelector` syntax, metrics are not collected from `collector` pods that are not running concurrently with a `logfilesmetricexporter` pod.
```

Comment:
There really is no direct relation to the collector pod here.  They are both daemonsets meaning they get rolled out to every node unless the scheduling says otherwise.  The implication is that you will not get metrics for any nodes upon which the LFME is missing